tools/clang_format.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import os
+import subprocess
+import typing
+
+
+def main(argv: typing.Optional[typing.List[str]] = None) -> int:
+    logging.basicConfig()
+    logger = logging.getLogger(__name__)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("files", nargs="*", help="only check these files")
+    args = parser.parse_args(argv)
+
+    cmd = ["git-clang-format"]
+
+    source_commit = os.environ.get("PRE_COMMIT_SOURCE")
```

Comment:
I think this is called PRE_COMMIT_FROM_REF in current versions of pre-commit.

Reply:
This is the new v2.2.0 value the old value is still supported. 
Mine is 2.0.1